lib/clients/redis.js

```diff
@@ -0,0 +1,48 @@
+var util = require('util');
+var BaseClient = require('./base');
+var _ = require('lodash');
+
+function buildRedisKey(namespace, metricName) {
+  return [namespace, metricName].join(':');
+}
+
+/**
+ * A client that can send metrics data to a redis backend
+ * @param {Object} opts 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)